lib/web_ui/lib/src/engine/assets.dart

```diff
@@ -68,6 +68,7 @@ class AssetManager {
         throw AssetManagerException(url, target.status);
       }
 
+      html.window.console.warn('Caught ProgressEvent with target: $target');
```

Comment:
Let's also log `asset` and the resolved `url`. That would help us narrow down the place in the gallery code that triggers this.